workitem/field_definition_blackbox_test.go

```diff
@@ -6,32 +6,71 @@ import (
 	"testing"
 
 	"github.com/fabric8-services/fabric8-wit/resource"
-	. "github.com/fabric8-services/fabric8-wit/workitem"
+	"github.com/fabric8-services/fabric8-wit/workitem"
```

Comment:
what's wrong with the `dot` import ?

Reply:
For practicality reasons: It prevents VSCode from jumping to definition when you ctrl-click on y symbol.
For Go reasons: See [this review comment](https://github.com/golang/go/wiki/CodeReviewComments#import-dot):

> Except for this one case, do not use import . in your programs. It makes the programs much harder to read because it is unclear whether a name like Quux is a top-level identifier in the current package or in an imported package.

Reply:
ok